api/app/src/main/kotlin/packit/model/OutpackResponse.kt

```diff
@@ -1,9 +1,21 @@
 package packit.model
 
+import com.fasterxml.jackson.annotation.JsonInclude
+import com.fasterxml.jackson.databind.ObjectMapper
 import java.io.Serializable
 
 data class OutpackResponse<T>(
         val status: String,
         val data: T,
         val errors: Any?
 ) : Serializable
+
+data class ErrorResponse(val error: ErrorDetail)
```

Comment:
I think it's fine to have all response models in this file but can we rename the file from `OutpackResponse` to just `Responses`?

Reply:
I think in due course we'll want to also modify the success responses to return in the above format (the same as the `OutpackResponse` format) but that can wait for a separate PR